administrator/components/com_cpanel/src/Controller/DisplayController.php

```diff
@@ -74,8 +74,11 @@ public function addModule()
             $position = 'cpanel';
         }
 
-        $this->app->setUserState('com_modules.modules.filter.position', $position);
-        $this->app->setUserState('com_modules.modules.client_id', '1');
+        // Administrator
+        $clientId = 1;
```

Comment:
```suggestion
        $clientId = (\Joomla\CMS\Application\ApplicationHelper::getClientInfo('administrator', true))->id;
```

I think this should work but needs a test